src/sentry/static/sentry/less/base.less

```diff
@@ -56,16 +59,35 @@ h4 {
 }
 
 h5 {
-  font-size: 1.6rem;
+  font-size: 1.8rem;
 }
```

Comment:
We’ll likely need to change these default font-sizes at some point because they’re bonkers today but for now having them in one place is an improvement. 1.6rem was being overridden elsewhere so I chose 1.8 here instead.